raiden/tests/scenarios/mediated-transfers-scnenarios.yaml

```diff
@@ -0,0 +1,120 @@
+settings:
+  #notify: "raiden-scenario-basic@example.com"
+  ## either `fast`, `medium` or an integer (in gwei)
+  gas_price: "fast"
+
+token:
+  ## To re-use an existing token give an address
+  ## address: "0xdEA4ca73f2527eBBe876d4B79170BA2b64515E3C"
+
+  ## To deploy a new one remove the `address` key
+  ## Optionally give name and symbol, both will be autogenerated if not given
+  #name: "some name"
+  #symbol: ""
+
+nodes:
+  commands:
+    #start: "docker start raiden_{:02d}"
+    #stop: "docker stop raiden_{:02d}"
+    #kill: "docker kill raiden_{:02d}"
+    stop: "pkill -f '127.0.0.1:50{:02d}'"
+    kill: "pkill -9 -f '127.0.0.1:50{:02d}'"
+    start: "raiden --config-file raiden_{0:02d}.toml --api-address 127.0.0.1:50{0:02d}"
+  range:
+    first: 1
+    last: 4
+    template: "localhost:50{:02d}"
+  ## Alternatively give an explicit list:
+  #list:
+  #  - node1
+  #  - node2
+  #  - node3
+
+## Available task types:
+## - serial
+## - paralle
+## - open_channel
+## - close_channel
+## - deposit
+## - transfer
+## - stop_node
+## - kill_node
+## - start_node
+## - assert
+## - wait
+
+
+scenario:
+  serial:
+    tasks:
+      - parallel:
+          tasks:
+          # A - B - C - D with 100 tokens in each channel
+          - open_channel: {from: 0, to: 1, total_deposit: 100}
+          - open_channel: {from: 1, to: 2, total_deposit: 100}
+          - open_channel: {from: 3, to: 2, total_deposit: 100}
+      - parallel:
+          tasks:
+          # assert A - B -C - D
+          - assert: {from: 0, to: 1, total_deposit: 100, balance: 100, state: "opened"}
+          - assert: {from: 1, to: 0, total_deposit: 0,  balance: 0,  state: "opened"}
+          - assert: {from: 1, to: 2, total_deposit: 100, balance: 100, state: "opened"}
+          - assert: {from: 2, to: 1, total_deposit: 0,  balance: 0,  state: "opened"}
+          - assert: {from: 3, to: 2, total_deposit: 100,  balance: 100,  state: "opened"}
+          - assert: {from: 2, to: 3, total_deposit: 0, balance: 0, state: "opened"}
+
+      - serial:
+          tasks:
+          # In A-B-C with 100 tokens in each channel, send 1 token from A to C
+          - transfer: {from: 0, to: 2, amount: 1}
```

Comment:
Can you please check for the ``expected_http_status: 200``?

Reply:
That is not necessary. The default is `2..` (i.e. any 2xx code is accepted).